src/queries/Discussions/GetModelPlanDiscussions.ts

```diff
@@ -17,6 +17,9 @@ export default gql`
         createdDts
         status
         isAssessment
+        createdByUser {
+          commonName
+        }
```

Comment:
@garyjzhao With the introduction of this property, do we still need to query for `collaborators` in this query, or can it be removed?

Reply:
@ClayBenson94 we do not need to query `collaborator` anymore, since we changed the way we are pulling the names due to this property. 

I will remove it from the query in an upcoming commit.